### PR TITLE
Doc

### PR DESCRIPTION
**Pull request by makotonakai**
## About 

Write the setup procedure of locksmith in README.md

## Issue

#5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,88 @@
...
+
+#### 1. Click "Create a trust anchor"
+![trust-anchor](/images/trust-anchor.png)
+
+#### 2. Paste your Root CA key to External certificate bundle
+![create-trust-anchor](/images/create-trust-anchor.png)
+
+
+#### 3. Create a special IAM role for IAM Roles Anywhere
+```
```
Comment (takutakahashi) [inline README.md:35]: Reccomend to explain what this json is. ex: assume-role-policy, or inline-policy

File: README.md [hunk 2]
```diff
@@ -1,2 +1,88 @@
...
+![profile](/images/profile.png)
+
+#### 6. Create a docker image for locksmith
+```
+cd locksmith
+export AWS_TRUST_ANCHOR_ARN=(ARN for your trust anchor)
+export AWS_PROFILE_ARN=(ARN for your AWS profile)
+export AWS_ROLE_ARN=(ARN for the AWS role that you are going to assume)
+export AWS_REGION=(your AWS region)
+docker compose up -d
```
Comment (takutakahashi) [inline README.md:67]: It hard to know this command is for build. Needs some comment.

File: README.md (hunk 1 above)
Comment (makotonakai) [reply README.md:36]: done

File: README.md (hunk 2 above)
Comment (makotonakai) [reply README.md:73]: done

**Review by takutakahashi — APPROVED** [review]
LGTM!

**Comment by makotonakai** [comment]
Thank you so much for your help,  @takutakahashi -san!

**Pull request merged**